src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * 
+ * A subclass of the Task Object. 
+ * Includes a deadline date to complete task by.
+ * 
+ */
```

Comment:
There is no need for empty line before and after javadoc. See example [here](https://se-education.org/guides/conventions/java/intermediate.html#comments)
```suggestion
/**
 * A subclass of the Task Object. 
 * Includes a deadline date to complete task by.
 */
```